tmt/export.py

```diff
@@ -325,16 +324,30 @@ def export_to_nitrate(test):
             if general:
                 try:
                     general_plan = find_general_plan(component)
-                    echo(style(
-                        f"Linked to general plan '{general_plan}'.",
-                        fg='magenta'))
-                    if not dry_mode:
-                        nitrate_case.testplans.add(general_plan)
+                    if general:
```

Comment:
why this line? looked at line 324 there is already `if general: ` . so that it has to be true everytime

Reply:
good point